dttools/src/catalog_update.c

```diff
@@ -127,15 +118,11 @@ int main(int argc, char *argv[]) {
 
 	char *text = jx_print_string(j);
 
-	char address[DATAGRAM_ADDRESS_MAX];
-	if (domain_name_cache_lookup(host, address)) {
-		datagram_send(d, text, strlen(text), address, port);
-	} else {
-		fatal("unable to lookup address of host: %s", host);
+	if(catalog_query_send_update(host, text) < 1) {
+		fatal("Unable to send update");
```

Comment:
Avoid `fatal()` unless we are really deep into something with no recovery -- it gives a rather disruptive error message.  `fprintf(stderr)` is usually the right thing to do in `main`